README.md

```diff
@@ -0,0 +1,13 @@
+# pac-cmd
+
+A command line tool to change proxy auto-config settings of operation system. Binaries included. Simply `make` to build it.
+
+# Usage
+
+```sh
+pac [on  <pac url> | off]
+```
+
+*  **Mac**: Have to `sudo` as setting pac is an privileged action.
+
+*  **Windows**: Install [MinGW-W64](http://sourceforge.net/projects/mingw-w64) to build pac, as it has up to date SDK headers we require.
```

Comment:
Oh that's simply because I had my Win 8.1 VM setup to write cgo before you tried build DLL through VC++. cgo doesn't support `cl`.

I prefer command line tools, it's smaller to download and install, quicker to launch, and also works cross-platform. But we can leave it later, when we want to build windows and linux dll with same process.